specification/marketplaceordering/resource-manager/Microsoft.MarketplaceOrdering/stable/2015-06-01/Agreements.json

```diff
@@ -500,20 +551,30 @@
       },
       "description": "Old Agreement Terms definition"
     },
+    "UnsupportedMediaTypeErrorResponse": {
+      "description": "Error response that indicates the media type in the request is unsupported.",
+      "type": "object",
+      "properties": {
+        "Message": {
+          "type": "string",
+          "description": "The details of the error."
+        }
+      }
+    },
```

Comment:
This one does not fit the ARM contract (but I guess you mentioned that this is not going to be changed anyway).

Reply:
Correct, unfortunately the code cant be changed, and the swagger must match the code.